rpp-charter.md

```diff
@@ -22,7 +22,8 @@ using HTTPS and JSON.
 Industry experience and use cases from EPP may influence the outputs
 of this working group, but direct compatibility of RPP with EPP is not
 a goal. Likewise, extensions or other changes for EPP are out of scope
-of this working group.
+of this working group. The group shall consider and enable migration scenarios
```

Comment:
This is the same comment as I made on the extensibility PR. Is "shall consider" too strong? Would "may consider" be better?

Reply:
in this case I can go with "may"